docs/architecture/PROCESS.md

```diff
@@ -0,0 +1,68 @@
+# ADR Creation Process
+
+1. Copy the `adr-template.md` file. Use the following filename pattern: `adr-next_number-title.md`
+
+2. Create a draft Pull Request if you want to get an early feedback.
+
+3. Make sure the context and a solution is clear and well documented.
+
+4. Add an entry to a list in the [README](./README.md) file.
+
+5. Create a Pull Request to propose a new ADR.
+
+## ADR life cycle
+
+ADR creation is an **iterative** process. Instead of trying to solve all decisions in a single ADR pull request, we MUST firstly understand the problem and collect feedback through a GitHub Issue.
```

Comment:
```suggestion
ADR creation is an **iterative** process. Instead of trying to solve all decisions in a single ADR pull request, we MUST initially understand the problem and collect feedback by having conversations in a GitHub Issue.
```